cloudbuild.yaml

```diff
@@ -12,17 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Builds docker image for gcp-variant-transforms:
-# Run using:
-# $ gcloud container builds submit --config cloudbuild.yaml --timeout 1h .
-substitutions:
-  _CUSTOM_TAG_NAME: 'latest'
+# This is used by Google Cloud Build GitHub app, which automatically builds the
+# code each time you push code to Github. This is what it does:
+# * Builds an image using the GitHub commit hash as the tag.
+# * Pushes that image to Container Registry.
+# * Runs small integration tests (only one test included for now) from that image.
 steps:
   - name: 'gcr.io/cloud-builders/docker'
     args:
       - 'build'
-      - '--tag=gcr.io/$PROJECT_ID/gcp-variant-transforms:${_CUSTOM_TAG_NAME}'
+      - '--tag=gcr.io/${PROJECT_ID}/gcp-variant-transforms:${COMMIT_SHA}'
```

Comment:
How does PROJECT_ID get set here?

Reply:
I think Cloud Build provides some [default substitutions](https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values), and `PROJECT_ID` is one of them. During the [setup](https://cloud.google.com/cloud-build/docs/run-builds-on-github), it will connect Google Cloud Build GitHub app to one GCP project.